scripts/install-contracts/index.js

```diff
@@ -0,0 +1,557 @@
+      }),
+      payment: "365125380",
+    },
+    {
+      name: "AdminContract->add_to_whitelist(SlashingVoterContract)",
+      contract: adminContract,
+      entrypoint: "add_to_whitelist",
+      args: RuntimeArgs.fromMap({
+        address: stringToCLKey(slashingVoterContract.contractHash),
+      }),
+      payment: "332336090",
+    },
+    // 8) RepoVoterContract <- SimpleVoterContract
+    //    ReputationContract <- SimpleVoterContract
+    {
+      name: "RepoVoterContract->add_to_whitelist(SimpleVoterContract)",
+      contract: repoVoterContract,
+      entrypoint: "add_to_whitelist",
+      args: RuntimeArgs.fromMap({
+        address: stringToCLKey(simpleVoterContract.contractHash),
+      }),
+      payment: "332353080",
+    },
+    {
+      name: "ReputationContract->add_to_whitelist(SimpleVoterContract)",
+      contract: reputationContract,
+      entrypoint: "add_to_whitelist",
+      args: RuntimeArgs.fromMap({
+        address: stringToCLKey(simpleVoterContract.contractHash),
+      }),
+      payment: "336074520",
+    },
+    // 9) ReputationContract <- BidEscrowContract
+    //    VaNftCo?tract <- BidEscrowContract
+    //    SlashingVoterContract <- (update_bid_escrow_list) <- BidEscrowContract
+    {
+      name: "ReputationContract->add_to_whitelist(BidEscrowContract)",
+      contract: repoVoterContract,
+      entrypoint: "add_to_whitelist",
+      args: RuntimeArgs.fromMap({
+        address: stringToCLKey(bidEscrowContract.contractHash),
+      }),
+      payment: "332353080",
+    },
+    {
+      name: "VaNftContract->add_to_whitelist(BidEscrowContract)",
+      contract: vaNFTContract,
+      entrypoint: "add_to_whitelist",
+      args: RuntimeArgs.fromMap({
+        address: stringToCLKey(bidEscrowContract.contractHash),
+      }),
+      payment: "337962950",
+    },
+    {
+      name: "SlashingVoterContract->update_bid_escrow_list([BidEscrowContract])",
+      contract: slashingVoterContract,
+      entrypoint: "update_bid_escrow_list",
+      args: RuntimeArgs.fromMap({
+        bid_escrows: CLValueBuilder.list([stringToCLKey(bidEscrowContract.contractHash)]),
+      }),
+      payment: "114789450",
+    },
+    // 10) ReputationContract <- OnboardingRequestContract
+    //     VaNftContract <- OnboardingRequestContract
+    {
+      name: "ReputationContract->add_to_whitelist(OnboardingRequestContract)",
+      contract: reputationContract,
+      entrypoint: "add_to_whitelist",
+      args: RuntimeArgs.fromMap({
+        address: stringToCLKey(onboardingRequestContract.contractHash),
+      }),
+      payment: "336074520",
+    },
+    {
+      name: "VaNftContract->add_to_whitelist(OnboardingRequestContract)",
+      contract: vaNFTContract,
+      entrypoint: "add_to_whitelist",
+      args: RuntimeArgs.fromMap({
+        address: stringToCLKey(onboardingRequestContract.contractHash),
+      }),
+      payment: "337962950",
+    },
+  ];
```

Comment:
Similarly this configuration could be moved to the configuration from the code. The configuration could look like:
```
   "private_key_path": "/Users/username/path/to/secret_key.pem",
   "node_rpc_uri": "http://localhost:11101/rpc",
   "contracts": {
     "VotingAssociateNFT": {
        "installCost": 1000000,
        "args": {
           "name": "VaNFTCRDao",
           "symbol": "VACRDAO",
           "base_uri": "http://example.com"
        }
     },
     // ...
     "Reputation": {
        "installCost": 1000000,
        "whitelistedContracts": [
          "SlachingVoter",
          "OnboardingRequest"
        ],
        "whitelistingCost": 100000
     }
   },
   // ...
 }
```

This way it also will be easier to verify that whitelisting was configured properly.